scripts/install_sc_machine_submodule.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+CURRENT_DIR=$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd)
+source "${CURRENT_DIR}/formats.sh"
+source "${CURRENT_DIR}/clone_project.sh"
+
+if [[ -z "${PLATFORM_PATH}" || -z "${SC_MACHINE_REPO}" || -z "${SC_MACHINE_BRANCH}" ]];
+then
+  source "${CURRENT_DIR}/set_vars.sh"
+fi
+
+cd "${PLATFORM_PATH}"
+
+clone_project "${SC_MACHINE_REPO}" "${SC_MACHINE_PATH}" "${SC_MACHINE_BRANCH}" "$1"
```

Comment:
You need point keys for each argument here

Reply:
--repo "${SC_MACHINE_REPO}" --path "${SC_MACHINE_PATH}" --branch "${SC_MACHINE_BRANCH}" "$1"